### ISSUE TITLE
VaceEncodingBlock fails on extremely narrow input images (kernel size > input size)

### ISSUE THREAD

**Issue by livepeer-tessa** [daydreamlive/scope#557]
## Summary
The `VaceEncodingBlock` encounters a convolution error when processing input images that have been resized to extremely narrow dimensions (width as small as 2 pixels), which is smaller than the 3x3 convolution kernel.

## Error Details
Multiple ERROR-level logs from fal.ai backend (28 Feb 2026):

**Pipelines affected:**
- `krea-realtime-video` (majority of occurrences)
- `streamdiffusionv2`

**Error pattern:**
```
Error in block: (vace_encoding, VaceEncodingBlock)
Error details: Calculated padded input size per channel: (257 x 2). Kernel size: (3 x 3). Kernel size can't be greater than actual input size
```

And for streamdiffusionv2:
```
Calculated padded input size per channel: (513 x 2). Kernel size: (3 x 3). Kernel size can't be greater than actual input size
```

## Analysis
- The padded input has dimensions like `(257 x 2)` or `(513 x 2)` — the width dimension is only 2 pixels
- The 3x3 convolution kernel cannot process inputs smaller than 3 pixels in any dimension
- The 257/513 heights suggest standard resolutions (256+1 for padding or 512+1)
- The 2-pixel width is abnormally narrow, likely caused by:
  1. Extreme aspect ratio input images/videos
  2. Incorrect resize/crop operations upstream
  3. A bug in resolution calculation when switching aspect ratios

## Suggested Fix
1. Add input validation in `VaceEncodingBlock` to reject or pad inputs below minimum dimensions
2. Add a minimum dimension check earlier in the pipeline (e.g., in input preprocessing)
3. Consider logging a more descriptive warning when inputs are near the minimum size

## Sample Timestamps (28 Feb 2026 UTC)
- 09:14:34 - 09:14:39 (krea-realtime-video)
- 09:17:14 (streamdiffusionv2)
- 09:19:43 - 09:22:48 (krea-realtime-video)

## Environment
- Backend: fal.ai
- App ID: `3564d791-2447-4a41-9ab8-53909db8c657`
- Blocks affected: `VaceEncodingBlock`

**Comment by livepeer-tessa** [comment]
## New occurrence — 2026-03-15

Same error pattern observed again, this time with transposed dimensions (height=2 instead of width=2):

```
Error in block: (vace_encoding, VaceEncodingBlock)
Error details: Calculated padded input size per channel: (2 x 513). Kernel size: (3 x 3). Kernel size can't be greater than actual input size
```

**Pipeline:** `krea-realtime-video`
**Session:** `[2c7ce7e9]`
**Time:** 2026-03-15 10:55:00 – 10:59:33 UTC
**Occurrences:** 500+ in ~5 minutes (every ~234ms), same flood pattern as before

**fal.ai job details:**
- job_id: `5193400c-da0f-4eef-8bdd-dd0fdd26c1db`
- node_id: `ce7099c4-0da2-e85f-2b20-51355980f222`
- worker: `fal-jobs/GPU-H100/5193400c-da0f-4eef-8bdd-dd0fdd26c1db`
- app_id: `b34bc0cd-90e4-4898-8142-3c905ca0f13c` (prod)

**Stack trace (same call path as before):**
```
scope/core/pipelines/krea_realtime_video/pipeline.py:271 _generate
  → diffusers modular_pipeline.py:932 __call__
  → vace_encoding.py:199 __call__
  → vace_encoding.py:402 _encode_extension_mode
  → vace/utils/encoding.py:123 vace_encode_frames
  → wan2_1/vae/wan.py:139 encode_to_latent
  → wan2_1/vae/wan.py:159 _encode_with_cache
  → wan2_1/vae/modules/vae.py:380 forward
  → vae.py:163 self.resample(x)   ← fails here
  → torch.nn.modules.conv.Conv2d (3x3 kernel, height=2 input)
```

Note this occurrence has dimensions `(2 x 513)` — the height dimension is 2 pixels (vs previous report's `(257/513 x 2)` where width was 2). Same root cause, different aspect ratio: extremely tall/narrow input after VAE downsampling.

These 500+ floods occurred just before logging from this worker stopped entirely at 10:59:33 UTC. No fal.ai logs have been received from prod since that time (~3 days ago as of this report).
